test/support/fixtures/accounts_fixtures.ex

```diff
@@ -28,8 +28,8 @@ defmodule Recognizer.AccountsFixtures do
     password_changed_at = Map.get(attrs, :password_changed_at, NaiveDateTime.utc_now())
 
     hashed_password =
-      if password == "" do
-        ""
+      if password == nil do
```

Comment:
What do you think about changing this to `is_nil/1` which is a little more common in Elixir?